code/__DEFINES/misc.dm

```diff
@@ -470,3 +470,14 @@ GLOBAL_LIST_INIT(pda_styles, sortList(list(MONO, VT, ORBITRON, SHARE)))
 #define FALL_INTERCEPTED		(1<<0) //Stops the movable from falling further and crashing on the ground
 #define FALL_NO_MESSAGE			(1<<1) //Used to suppress the "[A] falls through [old_turf]" messages where it'd make little sense at all, like going downstairs.
 #define FALL_STOP_INTERCEPTING	(1<<2) //Used in situations where halting the whole "intercept" loop would be better, like supermatter dusting (and thus deleting) the atom.
+
+//Religion
+
+#define HOLY_ROLE_PRIEST 1 //default priestly role
+#define HOLY_ROLE_HIGHPRIEST 2 //the one who designates the religion
+
+#define ALIGNMENT_GOOD "good"
+#define ALIGNMENT_NEUT "neutral"
+#define ALIGNMENT_EVIL "evil"
+
+#define SECT_TECH /datum/religion_sect/technology
```

Comment:
is this really needed? This starts bordering over defining imo. It's a type path

Reply:
yeah i didn't include this for my sect it just seems over the top

Reply:
Dont genetic mutations do type path defining too? (code/modules/_DEFINES/DNA.dm)

Reply:
i forgot why i added it